.github/workflows/push-1password-secrets-to-gha.yaml

```
@@ -0,0 +1,45 @@
+name: "Push secrets from 1Password to Github Actions"
+
+permissions:
+  id-token: write
+  contents: read
+
+on:
+  push:
+    branches:
+      - "push-secrets-*"
+
+jobs:
+  push-secrets:
+    name: "Push Secret"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Let's use the latest version and pin to hash like this:

```suggestion
      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Same for the other actions. We'll set Dependabot later.